### DISCUSSION TITLE
Background flashes creating a new window or tab

### DISCUSSION THREAD

**Discussion by edrozenberg** [ghostty-org/ghostty#3942] [Feature Requests, Ideas] [answered]
Ghostty Version 1.0.0
Build 8469
Commit https://github.com/ghostty-org/ghostty/commit/4b4d4062dfed7b37424c7210d1230242c709e990
(MacOS)

The background of the window or tab flashes when a new window or tab is created. Believe I've also sometimes seen a flash on closing a tab, but this is rare.

https://github.com/user-attachments/assets/5bcb2a07-ac70-43d2-b6d9-6726bcd04aab

I also looked at Terminal.app and Alactritty -
o Terminal.app never flashes
o Alacritty flashes but only when closing tabs or windows

**Comment by mitchellh** [comment, marked as answer]
Same issue as the other one you created. There is another issue covering this on resize, the same issue will fix both.
